### ISSUE TITLE
Visual label is not there for "Add task" button and  "Add project" button:A11y_.NET maui_User can get all the insights of Dashboard_Label & Instructions

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/maui#30810]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yTCS; #A11yMAS; #WCAG3.3.2; #Label & Instructions; #BM_.NETMaui_Win32_JUL2025; #DesktopApp; #Win32; #FTP; #A11ySev2; #.NET MAUI SDKs; #AILimited;

## Environment Details:
Application: Developer Balance (.NET Maui)
Windows 11
OS Build: 26100.2161
Version 24H2

## Repro Steps:

1. Install and open the Developer Balance app
2. TAB till "Add task" button
3. Tab till hamburger a and press ENTER 
4. TAB till Project and select it
5. TAB till "ADD project" button
6. Observe the issue

## Actual Result:
Visual label is not there for "Add task" button and  "Add project" button.
Note: Issue seen through out the application for "Add task" and "Add project" button. 

## Expected Result:
Visual label should be there for "Add task" button and  "Add project" button so user can under stand the purpose of button.

## User Impact:
User will not under stand the purpose of button they will get confuse with functionality of button.   

## Attachment
https://github.com/user-attachments/assets/2b1611a6-d676-4af8-9577-5e513ea4d9d0

### Attachment repro in MAC

https://github.com/user-attachments/assets/f20aa9fb-7bb2-4303-bf62-d077ad3ee109

**Comment by ghost** [comment]
This issue has been verified in Visual Studio 17.14.0 Preview 4.0 with MAUI versions (9.0.0 & 9.0.90). It can be reproduced on Windows platform.

**Comment by jsuarezruiz** [comment]
@sheiksyedm I noticed this has been reopened, could you share the current status? Let me know if there’s any way I can help.

**Comment by kapilvaishna** [comment]
Issue is repro in below version 

<img width="256" height="97" alt="Image" src="https://github.com/user-attachments/assets/df08ccb5-6e25-4899-9d01-9df0d8d3b48f" />

https://github.com/user-attachments/assets/44af23ba-530a-493d-a23d-3c252e4b97ab

**Comment by Vignesh-SF3580** [comment]
@kapilvaishna Verified with the .NET 10 template sample - when focusing on the Add button, the description is read properly by screen readers on both macOS and Windows. Attached a video for reference.

| MAC | Windows |
|----------|----------|
| <video width="300" height="600"  src="https://github.com/user-attachments/assets/c606ee39-6a33-4cc6-8b9a-63920ca926d5"> | <video width="300" height="600"  src="https://github.com/user-attachments/assets/7842a8c8-98c7-4eed-88c4-af00200e78d6">) |

**Comment by kapilvaishna** [comment]
@Vignesh-SF3580 Issue is not relates to announcement, issue is related visual label for controls.

**Comment by sheiksyedm** [comment]
@kapilvaishna This is a Floating Action Button (FAB) styled to represent the "Add Item" action using a "+" icon. Adding visual text to this button could negatively impact the design and visual balance.

@PureWeen  @davidortinau  could you please advise on how to proceed with this style? Specifically, should we retain the icon-only approach or adding the visual label to the button?

<img width="160" height="133" alt="Image" src="https://github.com/user-attachments/assets/76477e3d-5349-4d83-98b8-bbb8fb023b3e" />

**Comment by kapilvaishna** [comment]
@sheiksyedm Providing tool tip to icon will also work here.

**Comment by kapilvaishna** [comment]
+ icon can only tell some think to add not beyond it so this will not be an issue.
